12/umbraco-cms/reference/content-delivery-api/extension-points/README.md

```diff
@@ -0,0 +1,12 @@
+---
+description: A guide to extending the Content Delivery API.
+---
+
+# Extension points
+
+The Delivery API has been designed with extensibility in mind, offering multiple extension points that provide greater flexibility and customization options. This article explores these extension points, allowing you to tailor the API's behaviour and expand its capabilities to meet your specific requirements.
```

Comment:
"This article explores these extension points..."

Is something missing from this article?
It only list links, and doesn't seem to really explore anything? I might be wrong 🤔

Reply:
You are right! Maybe it doesn't make sense to be here as you suggested 🙂